src/mmw/js/src/user/views.js

```diff
@@ -279,8 +281,93 @@ var ForgotModalView = Marionette.ItemView.extend({
     }
 });
 
+var ItsiRegisterModalView = Marionette.ItemView.extend({
```

Comment:
Consider adding a model property here to designate which model this view is expecting. I've found this helpful but I'm not sure what others think.